pallets/weights/src/polymesh_contracts.rs

```diff
@@ -0,0 +1,54 @@
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 2.0.0
+
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::weights::{constants::RocksDbWeight as DbWeight, Weight};
+
+pub struct WeightInfo;
+impl polymesh_contracts::WeightInfo for WeightInfo {
+    fn put_code(l: u32, u: u32, d: u32) -> Weight {
+        (0 as Weight)
```

Comment:
I think the base weight is missing because the bench misattributes it to `u` and `d`. Perhaps `u` and `d` could be set to the maximum value?